content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -1239,6 +1239,49 @@ The following `ExecCredential` manifest describes a cluster information sample.
 {{% /tab %}}
 {{< /tabs >}}
 
+## Get self subject attributes
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+With the `Self Subject Attributes Review` API, you can be aware of user attributes represented by your current environment (kubeconfig file or in-cluster config).
+
+For convenience, the `kubectl auth who-am-i` command is present. Executing this command will produce the following output (yet different user attributes will be shown):
```

Comment:
Does this command also work with control plane versions before v1.26? If not, what happens when you try?